app/helpers/heatmap_helper.rb

```diff
@@ -166,7 +169,7 @@ def self.fetch_top_taxons(
     min_reads = MINIMUM_READ_THRESHOLD,
     sort_by = DEFAULT_TAXON_SORT_PARAM,
     threshold_filters = [],
-    species_selected = true
+    _species_selected = true
```

Comment:
Can we remove this line and change affected calls?

Reply:
Kept this parameter to use the flag to modify the query as we discussed.